modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/DatabaseNodeService.java

```diff
@@ -238,7 +240,7 @@ public void close() throws IOException {
         IOUtils.close(databases.values());
     }
 
-    void checkDatabases(ClusterState state) {
+    void checkDatabases(ClusterState state) { // cluster state listener
```

Comment:
What does the comment mean? Can you elaborate?

Reply:
Eh, it was just a reminder from myself to myself, I just removed it in a5906b3d63d.